themes/default/content/docs/pulumi-cloud/deployments/customer-managed-deployments.md

```diff
@@ -0,0 +1,81 @@
+title_tag: Get started with Customer-Managed Deployment Agents
+meta_desc: Here is a placeholder meta description of between 50 and 150 characters.
+menu:
+  pulumicloud:
+    parent: deployments
+    weight: 2
+---
+
+Customer-managed deployments allow you to self-host deployment agents bringing the same power and flexibility as Pulumi-hosted deployments. Self-hosting your agents comes with many benefits:
+
+- **Host anywhere**: You can host the deployment agents anywhere to manage infrastructure, even within your fully private VPCs
+- **Any hardware, any environment<sup>1</sup>**: Run the agents on any hardware of your choice and configure the environment that meets your needs
+- **Mix & Match**: You can use standard Pulumi-hosted deployments for your development and public stacks, and use self-hosted Customer-Managed Agents for your private network infrastructure. You can mix and match to suit your unique needs
+- **Multiple pools**: You can set up multiple deployment agent pools, assign stacks to specific pools, and scale agents dynamically to increase your deployment concurrency. Customers can have up to 150 concurrent deployments
+- **Meet Compliance**: You can configure the agents with the credentials needed to manage your infrastructure. This way your cloud provider credentials never leave your private network
+
+<sup>1</sup> *Currently Linux and MacOS are supported*
+
+{{% notes "info" %}}
+Customer-managed deployment agents are available to business-critical customers. Contact sales if you are interested and want to enable customer-managed deployments.
+{{% /notes %}}
+
+Before you begin, ensure you have installed [Pulumi Github App](/docs/using-pulumi/continuous-delivery/github-app/) and update the [source control settings](/docs/pulumi-cloud/deployments/get-started/#:~:text=Fill%20out%20the%20Source%20control%20settings). [Docker](https://docs.docker.com/engine/) is a requirement for the agent.
+
+## How to install and configure the agent
+
+Go to **Deployment runners** under settings and click on Add a new pool. Provide a name and description. You will get a new access token for the pool. Ensure to copy and save the token in a secure location.
+
+{{< video title="Install the agents" src="../customer-managed-deployment-agents-assets/install-the-agents.mp4" autoplay="true" loop="true" >}}
+
+{{< notes type="warning" >}}
+If you run into issues, ensure docker is running before you configure and start the agents. ensure you restart or refresh the shell session (example: `source ~/.zshrc`).
+{{< /notes >}}
+
+{{< video title="Configure the agents" src="../customer-managed-deployment-agents-assets/configure-the-agents.mp4" autoplay="true" loop="true" >}}
+
+Now your first agent should be up and running. You can verify the status of the agent by refreshing the page. Additionally, you can use the same token to set up multiple agents to increase concurrency of your deployments and they will be assigned to the same pool.
```

Comment:
should the doc include the code to install, configure and run? (same we added in the console)

Reply:
I think this is fine as our steps within the deployments runner page are pretty good..